dotnet/test/Azure.Iot.Operations.Services.UnitTests/AssetAndDeviceRegistry/ModelsConverterTests.cs

```diff
@@ -782,7 +788,9 @@ public void Device_ToModel_ShouldConvertAllProperties()
             Assert.Equal("1.0", mqttEndpoint.Version);
             Assert.Equal("mqtt", mqttEndpoint.EndpointType);
             Assert.NotNull(mqttEndpoint.AdditionalConfiguration);
-            Assert.Equal("mqtt-config-value", mqttEndpoint.AdditionalConfiguration);
+            string actualJson = JsonSerializer.Serialize(mqttEndpoint.AdditionalConfiguration);
+            string expectedJson = JsonSerializer.Serialize(JsonDocument.Parse("{\"someKey\":\"someValue\"}"));
```

Comment:
Also manually typing the same strings in both the arrangement and assertion sections increases the risk of typos and also might require uptae if values change

Reply:
I can refactor this, sure